packages/valaxy/src/types/config.ts

```diff
@@ -182,6 +186,16 @@ export interface ValaxyConfig<T = ValaxyThemeConfig> {
     }[]
   }
 
+  /**
+   * cover
+   * @description 封面
+   */
+  cover: {
```

Comment:
We don't need to set a global switch for cover, I think as long as there is a cover field in front-matter and then enable it, it is not enabled by default.